src/test/java/htsjdk/samtools/CRAMTestUtils.java

```diff
@@ -0,0 +1,64 @@
+package htsjdk.samtools;
+
+import htsjdk.samtools.cram.ref.CRAMReferenceSource;
+import htsjdk.samtools.cram.ref.ReferenceSource;
+import htsjdk.samtools.reference.InMemoryReferenceSequenceFile;
+import htsjdk.samtools.seekablestream.SeekableStream;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Iterator;
+
+public final class CRAMTestUtils {
+
+    private CRAMTestUtils(){};
```

Comment:
I understand that there's no point in instantiating this class, but why bother enforcing it with a private constructor?

Reply:
It does make it clear that there is no point in instantiating the class, which I like.

Reply:
Unlike a singleton, there's no harm if a user decides to instantiate it. They can't break anything or change behavior that we'd like to control. So I'd still say there is no need to bother enforcing it for this class.